src/message/messageActionSheet.js

```diff
@@ -400,7 +416,12 @@ export const showMessageActionSheet = (
   |},
   params: ConstructSheetParams<>,
 ): void => {
-  const optionCodes = constructNonHeaderActionButtons(params);
+  const optionCodes = constructNonHeaderActionButtons({
```

Comment:
Yeah, I meant that as shorthand for "can't use in react-native land without relying on a type that appears to be designed to be used primarily by the webview code"

I've reworded the commits to be more precise about what the actual thing I'm trying to avoid here is.